scripts/build

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -e
+
+flutter build apk --release
+
+VERSION=$(grep 'version:' pubspec.yaml | awk '{print $2}')
+mv "build/app/outputs/flutter-apk/app-release.apk" "build/Proxima-v$VERSION.apk"
```

Comment:
Good idea to move the .apk to a more accessible folder and add the version in the name!